### ISSUE TITLE
Can't connect to Vulkan application

### ISSUE THREAD

**Issue by UditDey** [GPUOpen-Tools/radeon_developer_panel#36]
I am attempting to use RDP to profile my Vulkan application on Arch Linux, but so far RDP does not detect any running applications and the documentation has not helped me.

Some system information:
Arch Linux
Hybrid graphics laptop
AMD Ryzen 5 4600H with Radeon integrated GPU
Nvidia GTX 1650ti discrete GPU
`lsmod` confirms I'm using the `amdgpu` driver

I am trying to profile a Vulkan application that is using the AMD integrated GPU. I can confirm that the AMD GPU is the one in use, and yet RDP's applications tab is empty.

I even tried it with the test Vulkan cube app `vkcube --gpu_number 1` (the `--gpu_number 1` ensure that the AMD integrated GPU is selected) but RDP still doesn't see anything running.

I've tried adding `vkcube` to the applications list with advanced mode turned on, but it always says that its offline even though `vkcube` is running:
![image](https://github.com/GPUOpen-Tools/radeon_developer_panel/assets/44467986/d5180a72-85bd-4ea2-abe2-f887e4216a22)

I'm not sure what to do now. For comparison, profiling applications using the Nvidia discrete GPU works just fine using Nvidia Nsight. I need to profile performance with the AMD integrated GPU as well but RDP is giving me nothing at all right now.

**Comment by matusnovak** [comment]
I am also having the exact same problem on `Manjaro Linux` with `AMD Radeon RX 6800 XT (RADV NAVI21)` and any Vulkan application. It works fine (with the same application running Vulkan) if I try it on Windows.

**Comment by PasVegan** [comment]
Same  with RDP v3.0.0.18 using Radeon 680M.

I have these warnings:
(14:17:51.736) WARNING [RRA Trace Source] System info received and parsed, but system does not support RRA.
(14:17:51.736) WARNING [RGD Trace Source] System info received and parsed, but system does not support RGD.

**Comment by rnvannatta** [comment]
I encountered this issue too: it turns out that if you're running amdgpu instead of amdgpu-pro, you have to use environment variables when running your app to take captures: the developer panel gui is not supported.

First run setup.sh inside the tools.

* If you're running X11: `MESA_VK_TRACE=rgp MESA_VK_TRACE_TRIGGER=/tmp/trigger ./a.out` and press F1
* Otherwise: `MESA_VK_TRACE=rgp MESA_VK_TRACE_TRIGGER=/tmp/trigger ./a.out` and `touch /tmp/trigger` in a seperate terminal tab.
* On older MESA drivers `RADV_THREAD_TRACE=500 ./a.out` to capture after 500 frames

The rgp file is written to /tmp

There probably should be a big warning panel if you're running amdgpu.

**Comment by EriKWDev** [comment]
Yup, using the open source driver was the issue

**Comment by qht1003077897** [comment]
However, the RADV_THREAD_TRACE=500 or  MESA_VK_TRACE_TRIGGER=/tmp/trigger  environment variable can only capture 1 frame, and cannot continuously capture N frames.

**Comment by EriKWDev** [comment]
Yup, I miss being able to capture several consecutive frames. It is a big disadvantage with the status quo solution with the environment variables :/
